### DISCUSSION TITLE
Bridge specifications

### DISCUSSION THREAD

**Discussion by getairdrop24** [zkSync-Community-Hub/zksync-developers#362] [Feature requests]
### Network

zkSync Lite

### Provide a brief description of the feature you would like to see implemented.

![Screenshot 2024-02-21 150934](https://github.com/zkSync-Community-Hub/zksync-developers/assets/85836509/dda3e6ae-8a14-48c9-b7be-e5daaa0a56d6)

How can i classify  which bridge i used to deposite eth. because there are types of bridge Native and Third party bridge. 

### Who does this feature request benefit the most (users, developers, protocol, etc.)?

After numerous transitions, we are confused about which bridge I used to deposit to ETH.  so please add a badge icon that can show  we used offical bridge.

**Comment by gregmolina5** [comment]
To address this feature request, you can add a badge icon or label in the user interface indicating whether the deposit was made through the official zkSync bridge or a third-party bridge. This addition will provide clarity to users regarding the origin of their deposited ETH, reducing confusion and improving user experience.

**Comment by tiovladelfabs** [comment]
A badge icon indicating the use of the official bridge would help users identify the source of their ETH deposits in zkSync Lite.

**Comment by renemilnev** [comment]
Implementing a badge icon to classify whether a user used the official bridge or a third-party bridge for depositing ETH on zkSync Lite would greatly benefit users. This feature would provide clarity and help users track their transactions more accurately.

**Comment by queskim** [comment]
We can implement a feature that adds a badge icon to show which bridge was used to deposit ETH on zkSync Lite. This will benefit users who may be confused about which bridge they used after multiple transactions.
